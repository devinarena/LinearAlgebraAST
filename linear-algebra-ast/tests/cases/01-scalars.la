print 3;
print 5.0;
print 1 + 1;
print 1.5 + 2.8;
print 1 * 2;
print 1.5 * 3.6;
print 1 / 2;
print 53.5 / 2.5;
print 1 - 2;
print 1.8 - 0.5;
print 2*3^2+5;
print 2^(3+5);