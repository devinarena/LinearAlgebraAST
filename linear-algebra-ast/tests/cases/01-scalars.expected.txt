TOKEN_PRINT - print - 1
TOKEN_NUMBER - 3 - 1
TOKEN_SEMICOLON - ; - 1
TOKEN_PRINT - print - 2
TOKEN_NUMBER - 5.0 - 2
TOKEN_SEMICOLON - ; - 2
TOKEN_PRINT - print - 3
TOKEN_NUMBER - 1 - 3
TOKEN_PLUS - + - 3
TOKEN_NUMBER - 1 - 3
TOKEN_SEMICOLON - ; - 3
TOKEN_PRINT - print - 4
TOKEN_NUMBER - 1.5 - 4
TOKEN_PLUS - + - 4
TOKEN_NUMBER - 2.8 - 4
TOKEN_SEMICOLON - ; - 4
TOKEN_PRINT - print - 5
TOKEN_NUMBER - 1 - 5
TOKEN_STAR - * - 5
TOKEN_NUMBER - 2 - 5
TOKEN_SEMICOLON - ; - 5
TOKEN_PRINT - print - 6
TOKEN_NUMBER - 1.5 - 6
TOKEN_STAR - * - 6
TOKEN_NUMBER - 3.6 - 6
TOKEN_SEMICOLON - ; - 6
TOKEN_PRINT - print - 7
TOKEN_NUMBER - 1 - 7
TOKEN_SLASH - / - 7
TOKEN_NUMBER - 2 - 7
TOKEN_SEMICOLON - ; - 7
TOKEN_PRINT - print - 8
TOKEN_NUMBER - 53.5 - 8
TOKEN_SLASH - / - 8
TOKEN_NUMBER - 2.5 - 8
TOKEN_SEMICOLON - ; - 8
TOKEN_PRINT - print - 9
TOKEN_NUMBER - 1 - 9
TOKEN_MINUS - - - 9
TOKEN_NUMBER - 2 - 9
TOKEN_SEMICOLON - ; - 9
TOKEN_PRINT - print - 10
TOKEN_NUMBER - 1.8 - 10
TOKEN_MINUS - - - 10
TOKEN_NUMBER - 0.5 - 10
TOKEN_SEMICOLON - ; - 10
TOKEN_EOF -  - 10
3
5
2
4.3
2
5.4
0.5
21.4
-1
1.3
